### DISCUSSION TITLE
Simulator for Windows

### DISCUSSION THREAD

**Discussion by Paul-Damian** [julianschill/klipper-led_effect#213] [Q&A] [answered]
Hello,
I have a hard time to find simulator for Windows. There is no link to it, Google doesn't help. Where is it? :-)
Best regards,
Paul

**Comment by julianschill** [comment, marked as answer]
Go to the Releases page https://github.com/julianschill/klipper-led_effect/releases and open the "Assets" field. There is a download to the simulator.

Here is a direct link: https://github.com/julianschill/klipper-led_effect/releases/download/v0.0.12/ledeffect-simulator-win.exe

**Reply by Paul-Damian** [reply]
Thank you very much!
